unifi_respondd/unifi_client.py

```diff
@@ -119,13 +119,14 @@ def get_ap_channel_usage(ssids, cfg):
 
 def get_location_by_address(address, app):
     """This function returns latitude and longitude of a given address."""
-    time.sleep(1)
     try:
         point = Point().from_string(address)
         return point.latitude, point.longitude
     except:
         try:
-            return app.geocode(address).raw["lat"], app.geocode(address).raw["lon"]
+            time.sleep(1)
+            geocode = app.geocode(address)
```

Comment:
we also should use cache for this:
https://operations.osmfoundation.org/policies/nominatim/

`Results must be cached on your side. Clients sending repeatedly the same query may be classified as faulty and blocked.`


or setup a own instance of nominatim:
https://nominatim.org/release-docs/latest/admin/Installation/